training/src/pages/ChildrenDemo/ChildrenDemo.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { ThemeProvider } from '@material-ui/core';
+import Calculation from '../../components/Math/Math';
+import Theme from '../../theme';
+
+export default class CalcultorDemo extends React.Component {
```

Comment:
CalcultorDemo not a valid component name.

Reply:
done